cnxpublishing/tests/views/test_admin.py

```diff
@@ -59,34 +67,33 @@ def test(self):
                 book = use_cases.setup_BOOK_in_archive(self, cursor)
                 db_conn.commit()
 
-                # Insert some data into post_publications.
-                cursor.execute("""\
-INSERT INTO post_publications
-    (module_ident, state, state_message)
-    SELECT
-        (SELECT module_ident FROM modules ORDER BY module_ident DESC LIMIT 1),
-           'Processing', '';""")
+                # Insert some data into the association table.
+                cursor.execute("""
+INSERT INTO document_baking_result_associations
+  (result_id, module_ident)
+SELECT
+  uuid_generate_v4(),
+  (SELECT module_ident FROM modules ORDER BY module_ident DESC LIMIT 1);""")
                 db_conn.commit()
 
                 cursor.execute("""\
-INSERT INTO post_publications
-    (module_ident, state, state_message)
-    SELECT
-        (SELECT module_ident FROM modules ORDER BY module_ident DESC LIMIT 1),
-           'Done/Success', 'Yay';
-""")
+INSERT INTO document_baking_result_associations
+  (result_id, module_ident)
+SELECT
+  uuid_generate_v4(),
+  (SELECT module_ident FROM modules ORDER BY module_ident DESC LIMIT 1);""")
 
         resp_data = self.target(request)
         self.assertEqual({
             'states': [
-                {'timestamp': resp_data['states'][0]['timestamp'],
+                {'created': resp_data['states'][0]['created'],
```

Comment:
This isn't really exposed anywhere. Right now this information goes into a template renderer. It's not likely for this to ever be exposed elsewhere. If it is we'll write something more robust.